websites/G/Granblue Fantasy/presence.ts

```diff
@@ -5,65 +5,151 @@ const presence = new Presence({
 	script = document.createElement("script"),
 	eventId = "PreMiD_GranBlueFantasy";
 
+enum Elements {
+	Plain,
+	Fire,
+	Water,
+	Earth,
+	Wind,
+	Light,
+	Dark,
+}
+
+const ElementIcons = {
+		// Plain element isn't really a thing for characters, but it's here for the sake of completion
+		[Elements.Plain]:
+			"https://cdn.rcd.gg/PreMiD/websites/G/Granblue%20Fantasy/assets/logo.png",
+		[Elements.Fire]:
+			"https://gbf.wiki/images/thumb/0/05/Icon_Element_Fire.png/20px-Icon_Element_Fire.png",
+		[Elements.Water]:
+			"https://gbf.wiki/images/thumb/8/80/Icon_Element_Water.png/20px-Icon_Element_Water.png",
+		[Elements.Earth]:
+			"https://gbf.wiki/images/thumb/9/94/Icon_Element_Earth.png/36px-Icon_Element_Earth.png",
+		[Elements.Wind]:
+			"https://gbf.wiki/images/thumb/6/62/Icon_Element_Wind.png/20px-Icon_Element_Wind.png",
+		[Elements.Light]:
+			"https://gbf.wiki/images/thumb/1/19/Icon_Element_Light.png/20px-Icon_Element_Light.png",
+		[Elements.Dark]:
+			"https://gbf.wiki/images/thumb/a/a5/Icon_Element_Dark.png/20px-Icon_Element_Dark.png",
```

Comment:
Images should be 512x512

Reply:
There aren't any larger images available, these are taken directly from game assets

Reply:
You can use a tool like waifu2x to upscale the image

Reply:
I am aware that you can upscale the images but that doesn't make the original assets look any better, especially for images already 25 times smaller in comparison

If you want 512x512 images for the *sake* of having 512x512 images then I can make them larger that way, but I don't see the point since they are of the same quality